inst/templates/ad_adlbhy.R

```diff
@@ -0,0 +1,88 @@
+# Name: ADLBHY
+#
+# Label: Lab Analysis Dataset for Hy's Law
+#
+# Input: adlb
+library(admiral)
+library(admiral.test) # Contains example datasets from the CDISC pilot project
+library(dplyr)
+library(lubridate)
+
+# Load source datasets ----
+
+# Use e.g. haven::read_sas to read in .sas7bdat, or other suitable functions
+# as needed and assign to the variables below.
+# For illustration purposes read in admiral test data
+# Using use_ad_template("adlb") and assigning the end object as admiral_adlb
```

Comment:
Could we provide link to the Hy's Law Guide and talk about this being a special dataset